lib/preprocess.js

```diff
@@ -281,7 +279,11 @@ export const getRetrievalResult = measurement => {
     case 904: return 'CANNOT_PARSE_CAR_FILE'
   }
 
-  if (measurement.status_code >= 300) return `ERROR_${measurement.status_code}`
+  if (measurement.status_code >= 300) {
+    const prefix = measurement.protocol === 'http' ? 'HTTP_' : 'LASSIE_'
+    // I cannot use + concatenation because TypeScript would complain
+    return `${prefix}${measurement.status_code}`
```

Comment:
```suggestion
    return `${prefix}ERROR_${measurement.status_code}`
```

Shall we keep `ERROR_` to clarify the meaning? Or is this always an error anyway?

Reply:
I guess it depends on how you define an error.

This function returns a "retrieval result code".

There is only one code for a retrieval that was successful - `OK`.

All other codes describe various failure modes.

That's why I think we don't need to include `ERROR` in those codes.